tests/commands/test_cmd_config.py

```diff
@@ -0,0 +1,73 @@
+# third-party imports
+from click.testing import CliRunner
+
+# local imports
+from shpkpr.cli import cli
+
+
+def test_no_args():
+    runner = CliRunner()
+    result = runner.invoke(cli, ['config'])
+
+    assert result.exit_code == 0
```

Comment:
How come the expected exit code in this case is `0`, when it's `2` in all the other `*_no_args_` cases?